src/components/Field/Field.tsx

```diff
@@ -70,14 +74,26 @@ export class Field extends React.PureComponent<FieldProps> {
     return (
       <div className={classes}>
         {label ? <Header sub>{label}</Header> : null}
-        <Input
-          value={value}
-          type={isAddress ? 'text' : type}
-          icon={icon}
-          loading={loading && !isAddress}
-          disabled={disabled}
-          {...rest}
-        />
+        {type === DATE_TYPE ? (
+          <SemanticDatepicker
+            value={value}
+            icon={icon ? icon : void 0}
+            loading={loading && !isAddress}
+            disabled={disabled}
+            format="DD-MM-YYYY"
```

Comment:
If it's the format that will used to show the date when used, if you don't set anything, will it comply with the native format of the browser? It would be nice if the component showed the date using the locale.

Reply:
The default format is YYYY-MM-DD as the figma was with DD-MM-YYYY I changed it but it will not adapt to locale if I remove it, it will display the default one